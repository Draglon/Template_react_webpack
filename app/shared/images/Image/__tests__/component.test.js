import React from 'react';

import HeaderComponent from '../component';

describe('<HeaderComponent /> component', () => {
  let component;
  const props = {
    className: 'some class to image',
    path: 'some path to image',
    alt: 'Alt to image',
  };

  it('Render property - with avatar', () => {
    component = shallow(<HeaderComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
