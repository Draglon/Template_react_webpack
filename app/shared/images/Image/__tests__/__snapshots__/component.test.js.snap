// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderComponent /> component Render property - with avatar 1`] = `
<img
  alt="Alt to image"
  className="some class to image"
  src="https://image.tmdb.org/t/p/original/some path to image"
/>
`;
